content/blog/2018-12-25-darktable-2.6/2018-12-25-darktable-2.6.md

```diff
@@ -0,0 +1,335 @@
+  </div>
+
+* To select a source, use Shift-click on the image. The cross is
+  placed at the mouse location and won't move until you select the
+  destination, with a simple click. This is interesting combined with
+  the permanent activation of the tool mentioned above: the source
+  for multiple strokes will be located at the same place, relative to
+  the destination.
+
+* A variant: use Control-Shift-Click instead of Shift-Click. This also
+  sets the source location, but this time it will remain fixed in
+  absolute coordinate instead of relative to the destination.
+
+### Split-frequency editing
+
+A common issue with photo retouching, typically for portrait, is that
+you want to hide spots, and sometimes reduce local contrast to make
+the skin appear softer, but keep the skin's texture. A brutal blur
+would make the skin overly soft and will give at best an "obviously
+post-processed" look. Let's take this image as an example (taken from
+a
+[pixls.us PlayRaw contest](https://discuss.pixls.us/t/playraw-hillbilly-portrait/5518):
+
+  <div style="text-align: center">
+![Example portrait]({filename}rt-overall-portrait.jpg)
+  </div>
+
+A common technique for this kind of retouching is to split the image
+into several images corresponding to different levels of details, and
+then combine the images together. This is what [GIMP's
+wavelet decompose plugin](https://pixls.us/articles/skin-retouching-with-wavelet-decompose/)
+does, for example. After splitting, one gets a blurry image
+corresponding to the coarse details, and one or several images
+containing details only. In our example, we get:
+
+  <div style="text-align: center">
+![Example portrait]({filename}rt-portrait-scale4.jpg)
+  </div>
+
+  <div style="text-align: center">
+![Example portrait, scale 5]({filename}rt-portrait-scale5.jpg)
+  </div>
+
+  <div style="text-align: center">
+![Example portrait, scale 6]({filename}rt-portrait-scale6.jpg)
+  </div>
+
+  <div style="text-align: center">
+![Example portrait, scale 7]({filename}rt-portrait-scale7.jpg)
+  </div>
+
+This kind of transformation is used internally by the "equalizer"
+module, which allows you to increase or decrease the importance of
+each level of detail in the image. While "equalizer" works globally on
+the image, "retouch" allows you to select the level of detail and the
+portion of image you want to work with.
+
+In the retouch module, this corresponds to the "wavelet decompose"
+part of the GUI:
+
+  <div style="text-align: center">
+![Wavelet decompose scales]({filename}rt-scales.png)
+  </div>
+
+This part shows one rectangle per scale (fine-grained on the left,
+coarse-grained on the right). The dark rectangle on the left
+corresponds to the full image, and the white one on the right to the
+residual image, i.e. the image where all other levels of details have
+been removed. By default, darktable always shows the final image, but
+you can visualize the details layers and residual image by clicking
```

Comment:
I like talking about "layers", but being consistent is good too.

I've added a sentence "Think of them as layers obtained from the source image, and re-combined together after retouching to obtain the final image." so that people like me get the wording layer, and I've `s/layer/scale/` elsewhere to be consistent.